libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx

```diff
@@ -7,17 +7,20 @@ import { GenericFormContext } from '../../providers';
 import { useContext } from 'react';
 
 /* eslint-disable-next-line */
```

Comment:
can be removed as well

Reply:
sure, will remove, just decided not to touch something that is not my change.